### DISCUSSION TITLE
Welcome to Hotel-Management-System Discussions!

### DISCUSSION THREAD

**Discussion by aliasar1** [aliasar1/Hotel-Management-System#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aneboy24** [comment]
hello how are you. i have been trying to clone your project but everytime i try to run it it brings me an errer that guna could not be found. can you kindly help out i am new at this cant solve it my self.
thanks
![Screenshot 2023-06-03 131643](https://github.com/aliasar1/Hotel-Management-System/assets/49230739/17820995-19a6-4120-b0c6-b7eebd17379f)

**Reply by aliasar1** [reply]
Hi. Already responded it here: https://github.com/aliasar1/Hotel-Management-System/issues/2 
Do let me know, if you need any further help.

**Comment by aneboy24** [comment]
i am new to this i dont know how to rectify the Guna problem

**Comment by innoxent911** [comment]
i am here to know about your system

**Comment by aliasar1** [comment]
Yeah! How can I help you?

**Reply by T7R67F877U** [reply]
What is the password to enter the program, my dear brother?

**Reply by aliasar1** [reply]
username: admin110 and password: hotels123

But before entering credentials, you need to run all the SQL from [HotelManagementDB.sql](https://github.com/aliasar1/Hotel-Management-System/blob/master/HotelManagementDB.sql) this file as it will generate schemas and tables accordingly along with adding admin to the database.

**Reply by T7R67F877U** [reply]
I did that and the program did not open and it shows me that the username or password is wrong
-If you don't mind, what is the answer to the security question to change the password?
What are the steps required of me to run the program without errors, because I have done a lot, but many errors appear. I have been working on your program for two weeks and I have not succeeded in running or opening it. What are the correct steps to run it, sir?

**Reply by aliasar1** [reply]
Have you connected database with the project?

**Reply by T7R67F877U** [reply]
yes connected, I always get this message: System.Data.SqlClient.SqlException: 'A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was unreachable. Check that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open the connection to SQL Server)'

**Reply by aliasar1** [reply]
[Solution](https://stackoverflow.com/questions/9945409/how-do-i-fix-the-error-named-pipes-provider-error-40-could-not-open-a-connec) - Follow this and make sure to test connection on Server to ensure its properly connected. Also you will need to update connection string of DB. Enable TCP/IP and Named Pipes from SQL Server Configuration Manager then restart the SQL Server service.

**Comment by danyal7693** [comment]
I get this error when running. please guide me
![2024-06-12_17-11-25](https://github.com/aliasar1/Hotel-Management-System/assets/86875041/d4eceaa1-129c-40e0-8ac7-e65ee6536426)

**Reply by aliasar1** [reply]
This happens because Visual Studio cannot find the .exe. Most likely, you just need to build (or rebuild) the project and make sure the correct startup project is selected.
